src/amo/css/styles.scss

```diff
@@ -1,3 +1,39 @@
 @import '~core/css/inc/mixins';
 @import '~ui/css/vars';
 @import './inc/vars';
+
+*,
+*::before,
+*::after,
+.addon .content .copy,
+.Addon-summary,
+.AddonDescription-contents,
+.AddonSummaryCard-header-text .AddonTitle,
+.AddonTitle,
+.AddonTitle .AddonTitle-author,
+.Card-header,
+.CollectionDetails-title,
+.CollectionDetails-description,
+.Definition-dd,
+.EditableCollectionAddon-name,
+.MetadataCard-list,
+.RatingManager legend,
+.Search .SearchResults-message,
+.SearchContextCard-header,
+.SearchResult-name,
+.ShowMoreCard-contents,
+.UserCollection-name,
+.UserProfile-biography,
+.UserReview,
+.UserReview-body.ShowMoreCard {
+  overflow-wrap: break-word;
+}
+
+.HeroRecommendation-heading,
+.SecondaryHero-message-description,
+.SecondaryHero-message-headline,
+.SecondaryHero-message-link,
+.SecondaryHero-module-description,
+.SecondaryHero-module-link {
+  overflow-wrap: anywhere;
```

Comment:
I think we should keep this specific rule in the different scss files. Maybe we don't need to change to `anywhere`, though. Having this list is useful, though.

Let's keep a backup of the list in this comment:

```css
.HeroRecommendation-heading,
.SecondaryHero-message-description,
.SecondaryHero-message-headline,
.SecondaryHero-message-link,
.SecondaryHero-module-description,
.SecondaryHero-module-link {
  overflow-wrap: anywhere;
}
``